### PR TITLE
Jenkins/language detection

### PR DESCRIPTION
**Pull request by osvegn**
Create job_dsl.groovy file with jobs creation.

Be able to detect a language using a bash script.

### CODE REVIEW

File: jenkins/job_dsl.groovy [hunk 1]
```diff
@@ -0,0 +1,31 @@
+job('Link-project') {
```
Comment (Cavonstavant) [inline jenkins/job_dsl.groovy:1]:
```suggestion
job('link-project') {
```

**Review by Cavonstavant — CHANGES_REQUESTED** [review]
You should separate the jenkins jobs in separate files: one for the base image builds and the other for the `link-project` job. This would avoid merge conflicts in the future and improve code clarity

File: jenkins/job_dsl.groovy (hunk 1 above)
Comment (Cavonstavant) [reply jenkins/job_dsl.groovy:1]: My comment also applies for each job in the `Whanos base images` folder :eyes:

File: jenkins/detect-language.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#/bin/bash
```
Comment (majent4) [inline jenkins/detect-language.sh:1]:
```suggestion
#!/bin/bash
```

File: jenkins/detect-language.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    fi
+    if [[ -f $language ]]; then
+        language_found=(${language_found[@]} ${available_languages[$index]})
+        language_index=$index
+    fi
+    let "index++"
+done
+unset language_found[0]
+
+if [[ ${#language_found[@]} > 1 ]]; then
```
Comment (majent4) [inline jenkins/detect-language.sh:27]: in Bash, > is used for string comparisons
```suggestion
if [[ ${#language_found[@]} -gt 1 ]]; then
```

**Review by majent4 — APPROVED** [review]
Ok, if the script fails it might be because you have to double quote variable expansions to prevent word splitting

**Review by Cavonstavant — APPROVED** [review]

**Pull request merged**